extra/onnx_ops.py

```diff
@@ -201,7 +201,10 @@ def Tile(input, repeats):
   final_shape = [r*s for r,s in zip(repeats_, input.shape)]
   return input.reshape(new_shape).expand(expand_shape).reshape(final_shape)
 
-def Range(start, limit, delta): return Tensor.arange(safe_numpy(start)[0], safe_numpy(limit)[0], step=safe_numpy(delta)[0])
+def Range(start, limit, delta):
+  start, limit, step = [int(safe_numpy(x)[0]) for x in (start, limit, delta)]
+  return Tensor.arange(start, limit, step)
```

Comment:
Could be `Tensor.arange(*[int(safe_numpy(x)[0]) for x in (start, limit, delta)])`